### ISSUE TITLE
Release v1.34.12

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#2186]
:robot: A dedicated issue to run tests and deploy release [v1.34.12](https://github.com/deckhouse/deckhouse/milestone/90).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.34.12 | test-v1.34.12*` or a branch `main | release-1.34`
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.19 | 1.20 | 1.21 | 1.22 | 1.23`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.34.12 | test-v1.34.12*` or a branch `main | release-1.34`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.21'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.19
/e2e/use/k8s/1.21

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.19 and 1.21 using image built from main branch.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.34
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [5230d2](https://github.com/deckhouse/deckhouse/commit/5230d2eab389ebff81cf74d80d82872c1ba70d95) in branch **main**:
```
[docs][deckhouse] Move section about collecting debug information to the FAQ (#2190)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [72df7a](https://github.com/deckhouse/deckhouse/commit/72df7aaa9aeff38e7b99445ea8d338c804884060) in branch **main**:
```
[node-manager] lock bashible-apiserver context generation on update (#2185)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3b1b84](https://github.com/deckhouse/deckhouse/commit/3b1b8430ba89e3d6b37540aedd76017345b9297a) in branch **main**:
```
Re-generate changelog v1.34.12 (#2193)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: konstantin-axenov <konstantin-axenov@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.34.12** is created.

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.34.12** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [73a9ea](https://github.com/deckhouse/deckhouse/commit/73a9eafdf2c4dde046c36b1b62b14abe537557e9) in branch **main**:
```
[candi] fix discover node ip bashible step (#2139)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [a987ff](https://github.com/deckhouse/deckhouse/commit/a987ffcb31db5e92a53d984dd7228dd559b1e158) in branch **main**:
```
[istio] don't export unready ingressgateway nodes via metadata-exporter (#2055)

Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
```
